app/assets/javascripts/discourse/ember-cli-build.js

```diff
@@ -56,40 +38,10 @@ module.exports = function (defaults) {
       enabled: true,
     },
     autoImport: {
-      forbidEval: true,
-      insertScriptsAt: "ember-auto-import-scripts",
-      watchDependencies: ["discourse-i18n"],
       webpack: {
-        // Workarounds for https://github.com/ef4/ember-auto-import/issues/519 and https://github.com/ef4/ember-auto-import/issues/478
-        devtool: isProduction ? false : "source-map", // Sourcemaps contain reference to the ephemeral broccoli cache dir, which changes on every deploy
         optimization: {
           moduleIds: "size", // Consistent module references https://github.com/ef4/ember-auto-import/issues/478#issuecomment-1000526638
         },
```

Comment:
Surprisingly, removing this `moduleIds` fix from the autoimport config means that our `script/check_reproducible_builds.rb` tool starts failing. So this config is somehow leaking through to the Embroider build 👀